.github/workflows/ci.yml

```
@@ -16,11 +16,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - run: rustup toolchain install stable --profile minimal
-
       - uses: Swatinem/rust-cache@v2
         with:
-          # bump this to invalidate build/dependency cache
+          # Bump this to manually invalidate the build/dependency cache
```

```suggestion
          # Bump this to manually invalidate the build/dependency cache.
```
It's easier to read full sentences.